static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -533,64 +539,100 @@ require.def(
             },
 
             _getClampedTimeForPlayFrom: function (seconds) {
+                if (this._currentPlayer === this.PlayerEmps.StreamingPlayer && !this._updatingTime) {
```

Comment:
Although we are not entirely comfortable with the logic being stated here, we are happy to accept this and monitor its impact on the applications (CATAL/TAP), with particular scrutiny on the updating of range mid buffering, the certification process will look at this.